mesonbuild/compilers/compilers.py

```diff
@@ -1095,6 +1100,13 @@ def get_assert_args(self, disable: bool, env: 'Environment') -> T.List[str]:
         """
         return []
 
+    def get_time64_args(self) -> T.List[str]:
```

Comment:
We usually implement these as something like:
```python
def get_time64_args(self, enable: bool) -> T.List[str]:
    return ['-D_TIME_BITS=64'] if enable else []
```
So that you don't have to have the check in every caller.